File: 1-TwoSum/Solution.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        visitedElements  = {}
        for index, number  in enumerate(nums):
            if number in visitedElements:
                return [visitedElements[number], index]
            else:
                visitedElements[target-number] = index

File: 1-TwoSum/Solution.java
class Solution {
  public int[] twoSum(int[] nums, int target) {
	Map<Integer, Integer> visited = new HashMap<>();
      
	for (int i = 0; i < nums.length; i++) {
        
	  if (visited.containsKey(nums[i])) {
        return new int[]{visited.get(nums[i]), i};
	  }
	  visited.put(target - nums[i], i);
	}
	return null;
  }
}
